plugins/ruby-foreman-statistics/debian/postinst

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# postinst script for foreman-*
+#
+# see: dh_installdeb(1)
+
+if [ ! -z "${DEBUG}" ]; then
+  set -x
+else
+  set -e
+fi
+
+. /usr/share/debconf/confmodule
+
+LOGFILE='/var/log/foreman-install.log'
+PLUGIN='foreman_statistics'
+
+# if this script aborts with an error dpkg can
+# hang if daemons have been started
+trap db_stop EXIT
+
+BUNDLE=bundle
+[ -h /usr/bin/foreman-ruby ] && BUNDLE="/usr/bin/foreman-ruby /usr/bin/bundle"
+
+# Update gems
+export HOME=/usr/share/foreman
+cd /usr/share/foreman
+if [ ! -z "${DEBUG}" ]; then
+  if [ -n "$2" ] ; then
+    $BUNDLE update $PLUGIN --local
+  else
+    $BUNDLE install --local --no-prune
+  fi
+else
+  if [ -n "$2" ] ; then
+    $BUNDLE update $PLUGIN --local 2>&1 >> $LOGFILE
+  else
+    $BUNDLE install --local --no-prune 2>&1 >> $LOGFILE
+  fi
+fi
+
+# DB migrate/seed after install
+if [ -f /usr/share/foreman/config/database.yml ]; then
+  if [ ! -z "${DEBUG}" ]; then
+    /usr/sbin/foreman-rake db:migrate || true
+    /usr/sbin/foreman-rake db:seed || true
```

Comment:
@ehelms didn't we drop these from the post installation? or did we only do that for rpms?